incipyt/tools/git.py

```diff
@@ -20,6 +20,9 @@ def __init__(self):
 
         signals.vcs_ignore.connect(self._slot)
 
+        self._set_git_credential_email = False
+        self._set_git_credential_name = False
```

Comment:
git doesn't use credentials, github does. user.name and user.email are not credentials

Reply:
a name that starts with a verb should be a function, not a boolean